scripts/test.ps1

```diff
@@ -219,6 +219,10 @@ if ($CodeCoverage) {
     }
 }
 
+if ($IsLinux) {
+    $Env:OPENSSL_CONF=/etc/ssl/openssl.cnf
```

Comment:
What's this do? Why is it needed?

Reply:
in CxPlatCryptInitalize calls OPENSSL_init_ssl with the OPENSSL_INIT_LOAD_CONFIG option, but since the openssl build for msquic doesn't specify or install a configuration file, none is found and the libcrypto init fails, causing several tests to fail.  This specifies that the config file to use is the system default

Reply:
Is this a new behavior? This would break existing callers that upgrade to the latest MsQuic (if it uses this openssl) and don't have this set.

Reply:
It shouldn't be.  I'm not sure when it was introduced, but certainly in by the early 3.0 series it was in place.  a quic search suggests we've been doing configuration file loading like this since 2020 (likely before that).  So I'm actually wondering why quictls is working in this way, unless they made some changes to support doing so.

It can also be fixed by modifying the OPENSSL_init_ssl call to not pass the LOAD_CONFIG flag, so we have a few options here

Reply:
I'm modifying this so that, instead of setting the variable, I'm building openssl with no-autoload-config and removing the LOAD_CONFIG setting from the call to OPENSSL_ssl_init().  Given that you're not shipping an openssl  .cnf file, that seem sensible.  Though I'm still at  a loss for how this is working at all with quictls, as it appears to have the same config load strategy without changes.